scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -19,6 +19,7 @@ workflow GvsBulkIngestGenomes {
         # Begin GvsAssignIds
         String dataset_name
         String project_id
+        Boolean samples_are_controls = false
```

Comment:
added this in order to ingest control samples into my quickstart

Reply:
ooooh does this mean we can load controls via bulk ingest now?!?!?

Reply:
In a separate run after creating a sample_set of control samples, yes.  Not sure if that does everything that https://broadworkbench.atlassian.net/browse/VS-1205 had in mind, tho.

Reply:
can we plz look at that ticket and update / close as appropriate based on the changes here?